doc/references.bib

```diff
@@ -2049,3 +2049,15 @@ @Article{Kappenman2021
 }
 
 @Comment{jabref-meta: databaseType:bibtex;}
+
+@article{Yao2019,
```

Comment:
For consistency, you could rename to `YaoEtAl2019` (compare to other references).

Reply:
Should be updated now!